7.2.0/h2/delete_um_tenant.sh

```
@@ -0,0 +1,89 @@
+
+# Iterate through common tenant IDs and delete them from UM_TENANT
+echo "Starting UM_TENANT deletion process..."
+for TENANT_ID in $COMMON_TENANTS; do
+  echo "Processing TENANT_ID=$TENANT_ID"
+
+  # Generate and execute the delete query
+  {
+    echo "BEGIN TRANSACTION;"
+    echo "DELETE FROM UM_TENANT WHERE UM_ID = $TENANT_ID;"
+    echo "COMMIT;"
+  } | java -cp "$H2_JAR_PATH" org.h2.tools.Shell -url "$DB_URL" -user "$DB_USER" -password "$DB_PASSWORD" 2>&1 > /tmp/sql_output.log
```

Shellcheck (0.11.0)

[warning] To redirect stdout+stderr, 2>&1 must be last (or use '{ cmd > file; } 2>&1' to clarify).

(SC2069)